test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,46 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const fixedPort = 8080;
```

Comment:
@sebastianplesciuc Oh, I think I see the test/comment you are referring to. In that case, other (intentional and for testing purposes) errors in the code prevent that port from ever being in use. If I understand what's going on in this test (and I may not!), that port (the one that is now `8080`) does in fact get used. A connection is attempted there and ECONNREFUSED is expected, meaning nothing is listening on that port. So if something else is using that port, bad things happen. Again, I may be misunderstanding the test, but that's the way it seems to me. (Massively divided attention right now, apologies if I'm hurting more than I'm helping by participating.)